app/models/forms/bulk_upload_sales/upload_your_file.rb

```diff
@@ -38,21 +38,28 @@ def save!
           filename: file.original_filename,
         )
 
-        if upload_enabled?
-          storage_service.write_file(bulk_upload.identifier, File.read(file.path))
-        end
+        storage_service.write_file(bulk_upload.identifier, File.read(file.path))
+
+        ProcessBulkUploadJob.perform_later(bulk_upload:)
 
         true
       end
 
     private
 
       def upload_enabled?
```

Comment:
Does this need its own method? Could we just call FeatureToggle.upload_enabled?

Reply:
I've removed this now. Not really too fussed if it's private implementation